baggage-processor/src/test/java/io/opentelemetry/contrib/baggage/processor/BaggageProcessorCustomizerTest.java

```diff
@@ -78,7 +78,7 @@ private static void assertCustomizer(
     }
 
     await()
-        .atMost(Duration.ofSeconds(1))
+        .atMost(Duration.ofSeconds(60))
```

Comment:
In instrumentation repo `waitAndAssertTraces` uses awaitility under the hood with default timeout of 10s. While 60s does the job I probably would have started with a lower timeout, like the default 10 or maybe 20.

Reply:
done